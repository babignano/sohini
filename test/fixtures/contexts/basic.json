{
	"headline": "The core software is built by hundreds of community volunteers",
	"img":      "http://www.example.com/image.jpg",
	"link":     "http://www.example.com/post/1"
}