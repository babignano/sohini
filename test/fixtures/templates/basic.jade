doctype html
html(lang="en")
  head
    title= headline
  body
    h1= headline