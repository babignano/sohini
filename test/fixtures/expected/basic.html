<!DOCTYPE html>
<html lang="en">
  <head>
    <title>The core software is built by hundreds of community volunteers</title>
  </head>
  <body>
    <h1>The core software is built by hundreds of community volunteers</h1>
  </body>
</html>